lib/messaging/messagehandler_message.rb

```diff
@@ -10,4 +10,9 @@ def initialize
     handled = false
   end
 
+  def handle_message(message)
+    message.handled = true
+    puts 'This is probably where the message gets handled.'
```

Comment:
What it actually does in the messagehandler_* objects is just putting stuff into the database. The stuff in controller.rb will read the database and execute it. If the simulation is set, it will not send the commands to the arduino but just print it on the screen and mark them as done.